packages/bpk-component-mobile-scroll-container/src/BpkMobileScrollContainer.js

```diff
@@ -140,35 +140,38 @@ class BpkMobileScrollContainer extends Component<Props, State> {
   }, 100);
 
   setScrollIndicatorClassName = () => {
-    const classNames = computeScrollIndicatorClassName(
-      this.scrollerEl,
-      this.props.leadingIndicatorClassName,
-      this.props.trailingIndicatorClassName,
-    );
-
-    if (!classNames) {
-      return;
-    }
-
-    this.setState(() => ({
-      scrollIndicatorClassName: classNames.join(' '),
-    }));
+    requestAnimationFrame(() => {
```

Comment:
computeScrollIndicatorClassName and computeScrollBarAwareHeight will get the height and width of the element, and frequent calls can lead to reflow. Therefore, move the computations to the last frame to improve performance and avoid reflow.